src/targets/gpu/target.cpp

```diff
@@ -60,6 +61,7 @@ std::vector<pass> target::get_passes(migraphx::context& gctx, const compile_opti
         normalize_ops{},
         decompose{},
         dead_code_elimination{},
+        optimize_qdq_format{},
```

Comment:
Can we name this `simplify_qdq`? That will be consistent in naming with the other simplify passes we have.